tools/cli/NearObjectCliUwbSessionEventCallbacks.cxx

```diff
@@ -6,31 +6,40 @@
 using namespace nearobject::cli;
 
 void
-NearObjectCliUwbSessionEventCallbacks::OnSessionEnded(::uwb::UwbSession* /* session */, ::uwb::UwbSessionEndReason /* reason */)
+NearObjectCliUwbSessionEventCallbacks::OnSessionEnded(::uwb::UwbSession* session, ::uwb::UwbSessionEndReason reason)
 {
-    std::cout << "Session Ended" << std::endl;
+    std::cout << "Session " << session->GetId() << ": Session Ended" << std::endl;

```

Comment:
I think it would be helpful to include a prefix indicating the number is a session id. It might also help to output it with a fixed width (8 characters, since that's the max size), and as hex. Eg.

```
"Session with id=0xaabbccdd ended"
```

For controlling the width and output base of numbers, see the `UwbMacAddress` parsing code.